README.md

```diff
@@ -1,182 +1,304 @@
-⚠️For any issue with since version OpenDTU v4.4.3 please update to latest code where this is fixed. OpenDTU changed the API⚠️
+# dbus-opendtu
```

Comment:
You seem to have restructured the document, so the diff is useless. But as far as I can see, I like it.
Could you try to shortly (!) summarize your changes?

Reply:
My intention was originally to add some notes and config about com.victronenergy.inverter. Then I lost control... :D 

- Fix all Markdownlint Errors and apply recommendations to the Readme
- Restructured the order of Topics in flavor what the user might want to see first. (Please discuss if you disagree) 
- Fix some small spelling mistakes
- Add Table of Contents
- Add a "greeting screenshot" for the project
- Removed or renamed some titles and modify content
- Add Content that I thought it is useful
 - Separate config table into 3 parts (default, inverter, template) 
 
 
 You can see the results here: [github](https://github.com/henne49/dbus-opendtu/blob/d171ef835346b054e6f23ad4f71a0c9c22fe5d29/README.md)

Reply:
Great! Thank you very much. @henne49 also?